build/run error:  Cannot find module '@/types/contracts'

## Bug description
when running the docker-compose, we get the following error:

![image](https://github.com/safe-global/safe-wallet-web/assets/8118992/92ee6d46-484c-430d-8df2-64ce6d6f9c7f)

indeed missing files are set on `.gitignore`

![image](https://github.com/safe-global/safe-wallet-web/assets/8118992/85c1337b-d169-42df-9901-fc755f9f483e)

wanted to understand if is that possible to get the missing libs in order to run this project
or how to possibly overcome this setup issue
thanks


## Environment
docker and docker-compose on osx

## Steps to reproduce
1. git clone
2. set env file
3. docker-compose up
4. `ui` service crashes, in the logs you get my screenshot

## Expected result
1. git clone
2. set env file
3. docker-compose up
4. voila! working

## Obtained result

````
./src/components/tx/ApprovalEditor/utils/approvals.ts:1:32
Type error: Cannot find module '@/types/contracts' or its corresponding type declarations.

> 1 | import { ERC20__factory } from '@/types/contracts'
    |                                ^
  2 | import { UNLIMITED_APPROVAL_AMOUNT } from '@/utils/tokens'
  3 | import type { BaseTransaction } from '@safe-global/safe-apps-sdk'
  4 | import type { DecodedDataResponse, TokenInfo } from '@safe-global/safe-gateway-typescript-sdk'

--- a/Dockerfile
+++ b/Dockerfile
@@ -5,6 +5,7 @@ COPY . .
 
 # install deps
 RUN yarn install --frozen-lockfile
+RUN yarn after-install
 
 ENV NODE_ENV production
 

